Assets/MixedRealityToolkit-SDK/Features/Audio/Influencers/AudioLoFiEffect.cs

```diff
@@ -0,0 +1,208 @@
+        }
+
+        private void OnValidate()
+        {
+            influencerController = gameObject.GetComponent<AudioInfluencerController>();
+        }
+
+        /// <summary>
+        /// Populates the source quality filter settings collection.
+        /// </summary>
+        private void LoadQualityFilterSettings()
+        {
+            if (sourceQualityFilterSettings.Keys.Count > 0) { return; }
+
+            sourceQualityFilterSettings.Add(
+                AudioLoFiSourceQualityType.FullRange,
+                new AudioLoFiFilterSettings(10, 22000));    // Frequency range: 10 Hz - 22 kHz
+            sourceQualityFilterSettings.Add(
+                AudioLoFiSourceQualityType.NarrowBandTelephony,
+                new AudioLoFiFilterSettings(300, 3400));    // Frequency range: 300 Hz - 3.4 kHz
+            sourceQualityFilterSettings.Add(
+                AudioLoFiSourceQualityType.WideBandTelephony,
+                new AudioLoFiFilterSettings(50, 7000));     // Frequency range: 50 Hz - 7 kHz
+            sourceQualityFilterSettings.Add(
+                AudioLoFiSourceQualityType.AmRadio,
+                new AudioLoFiFilterSettings(40, 5000));     // Frequency range: 40 Hz - 5 kHz
+            sourceQualityFilterSettings.Add(
+                AudioLoFiSourceQualityType.FmRadio,
+                new AudioLoFiFilterSettings(30, 15000));    // Frequency range: 30 Hz - 15 kHz
+        }
+
+        /// <summary>
+        /// Settings for the filters used to simulate a low fidelity sound source.
+        /// </summary>
+        /// <remarks>
+        /// This struct is soley for the private use of the AudioLoFiEffect class.
+        /// </remarks>
+        private struct AudioLoFiFilterSettings
+        {
+            /// <summary>
+            /// The frequency below which sound will be heard.
+            /// </summary>
+            public float LowPassCutoff
+            { get; private set; }
+
+            /// <summary>
+            /// The frequency above which sound will be heard.
+            /// </summary>
+            public float HighPassCutoff
+            { get; private set; }
+
+            /// <summary>
+            /// FilterSettings constructor.
+            /// </summary>
+            /// <param name="highPassCutoff">High pass filter cutoff frequency.</param>
+            /// <param name="lowPassCutoff">Low pass filter cutoff frequency.</param>
+            public AudioLoFiFilterSettings(float highPassCutoff, float lowPassCutoff) : this()
+            {
+                HighPassCutoff = highPassCutoff;
+                LowPassCutoff = lowPassCutoff;
+            }
+
+            /// <summary>
+            /// Checks to see if two FilterSettings objects are equivalent.
+            /// </summary>
+            /// <returns>True if equivalent, false otherwise.</returns>
+            public static bool operator ==(AudioLoFiFilterSettings a, AudioLoFiFilterSettings b)
+            {
+                return a.Equals(b);
```

Comment:
Boxing here. Did we implement our own equality check for `AudioLoFiFilterSettings` ?